.github/dependabot.yml

```diff
@@ -9,14 +9,98 @@ updates:
     directory: '/'
     schedule:
       interval: 'weekly'
+      time: '09:00'
+      day: 'tuesday'
     groups:
-      npm:
+      changesets:
         patterns:
-          - '*'
-    ignore:
-      - dependency-name: '*'
-        # ignore all major updates as we want to do them manually
-        update-types: ['version-update:semver-major']
+          - '@changesets/*'
+        update-types:
+          - 'minor'
+          - 'patch'
+      babel:
+        patterns:
+          - '@babel/*'
+          - 'babel-*'
+        update-types:
+          - 'minor'
+          - 'patch'
+      guardian:
+        patterns:
+          - '@guardian/*'
+        update-types:
+          - 'minor'
+          - 'patch'
+      types:
+        patterns:
+          - '@types/*'
+          - 'type-fest'
+          - 'tslib'
+        update-types:
+          - 'minor'
+          - 'patch'
+      trouble-makers:
+        patterns:
+          - '*eslint*'
+          - '*prettier*'
+        update-types:
+          - 'minor'
+          - 'patch'
+      playwright:
+        patterns:
+          - '@playwright/*'
+        update-types:
+          - 'minor'
+          - 'patch'
+      jest:
+        patterns:
+          - '*jest*'
+        update-types:
+          - 'minor'
+          - 'patch'
+      webpack:
+        patterns:
+          - '*webpack*'
+          - '*-loader'
+        update-types:
+          - 'minor'
+          - 'patch'
+      minor-dependencies:
+        dependency-type: 'production'
+        update-types:
+          - 'minor'
+          - 'patch'
+      minor-devDependencies:
+        dependency-type: 'development'
+        update-types:
+          - 'minor'
+          - 'patch'
+        exclude-patterns:
+          - '@changesets/*'
+          - '@babel/*'
+          - 'babel-*'
+          - '@guardian/*'
+          - '@types/*'
+          - 'type-fest'
+          - 'tslib'
+          - '*eslint*'
+          - '*prettier*'
+          - '@playwright/*'
+          - '*jest*'
+          - '*webpack*'
+          - '*-loader'
+      # We can have a PR for all major updates for production or development dependencies.
+      # if we want to be more granular we can create a group with update-type major for each dependency.
+      major-dependencies:
```

Comment:
I grouped major for dependencies and a separate group for devDependencies. I know Jake suggested to have a single major PR for each dependency but as far as I understand there is no way to do that unless we specify a group with for single dependency with major.

Reply:
If we omit group config for major dependencies, will it just continue to raise single PRs for each major dependency update?

From [Configure dependabot.yml > groups](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups): 

> If a dependency doesn't belong to any group, Dependabot will continue to raise single pull requests to update the dependency to its latest version as normal. GitHub reports in the logs if a group is empty. For more information, see "[Dependabot fails to group a set of dependencies into a single pull request](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/troubleshooting-dependabot-errors#dependabot-fails-to-group-a-set-of-dependencies-into-a-single-pull-request)."

Reply:
Yes you are right! Not sure how I missed that. Shall I remove the `major-dependencies` and `major-devDependencies` groups so we expect to have a single dependency PR for major?

Reply:
Yes, I think that would be a good starting place. We can always change it if it's too noisy 😊

Reply:
Ok sounds good. Addressed in 7f55a9881b0974a9504a54eeb923124199509ece